dev-python/shtab/shtab-1.6.5.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} pypy3 )
```

Comment:
Any reason not to support 3.10? It's kinda weird to have PyPy3.10 but not CPython 3.10 here.